libminifi/test/resources/TestControllerServices.yml

```diff
@@ -0,0 +1,62 @@
+Flow Controller:
```

Comment:
Can't leave comments on binary items so I am annotating on the nearest neighbor.  Noticed that libminifi/test/libminifi.a seems to have crept its way into the commit.  Could we update .gitignore to cover the new structure as well?